kafka-ballerina/tests/client_tests.bal

```diff
@@ -52,8 +52,8 @@ function consumerServiceTest() returns error? {
         clientId: "test-consumer-1"
     };
     Listener consumer = check new (consumerConfiguration);
-    var attachResult = check consumer.attach(consumerService);
-    var startResult = check consumer.'start();
+    Error? attachResult = check consumer.attach(consumerService);
```

Comment:
When we use the `check` expression, the `attachResult` cannot be an `Error` right?

Check the other places as well.

Reply:
Fixed in 9eacde4cd32cfd5eaa957c4a4646f046744e95a5